ui/src/app/components/html-viewer.tsx

```diff
@@ -6,12 +6,14 @@ import { WindowSizeProps } from 'app/utils';
 
 export interface Props extends WindowSizeProps {
   containerStyles?: React.CSSProperties;
+  onFirstScroll: () => void;
   onLastPage: () => void;
```

Comment:
need to keep the other event function because TOS uses it.  I wonder if TOS has the same problem?